docs/src/starknet/script.md

```diff
@@ -35,12 +35,21 @@ Some of the planned features that will be included in future versions are:
 - account creation/deployment
 - multicall support
 - dry running the scripts
-- init subcommand
 
 and more!
 
 ## Examples
 
+### Initialize a script
+
+To get started, a deployment script with all required elements can be initialized using the following command:
+
+```shell
+$ sncast script init my_script
+```
+
+For more details, see [init command](../appendix/sncast/script/init.md).
+
```

Comment:
nit: In a `Note` field, let's add more specific information about aht to add (and where) if we are working in a workspace. Scarb docs for workspaces could be linked in here to add more context, as well.